web/source/osk/visualKeyboard.ts

```diff
@@ -73,6 +69,37 @@ namespace com.keyman.osk {
       this._layerId = value;
     }
 
+    get currentLayer(): OSKLayer {
+      if(this.layerGroup && this.layerId) {
+        return this.layerGroup.layers[this.layerId];
+      }
```

Comment:
Currently not all code paths return a value?

```suggestion
      return this.layerId ? this.layerGroup?.layers[this.layerId] : null;
```